pkgs/os-specific/bsd/openbsd/pkgs/make-rules/package.nix

```diff
@@ -0,0 +1,28 @@
+{ mkDerivation, stdenv }:
+
+mkDerivation {
+  path = "share/mk";
+  noCC = true;
+
+  buildInputs = [ ];
+  nativeBuildInputs = [ ];
```

Comment:
Are these lines necessary?

Reply:
Yes because there are default deps to be cancelled.